### PR TITLE
chore: migrated to pnpm 

### PR DESCRIPTION
**Pull request by priyankarpal**
migrated to pnpm #372

### CODE REVIEW

**Review by rupali-codes — APPROVED** [review]
Thanks Priyankar, lgtm

File: README.md [hunk 1]
```diff
@@ -51,74 +53,80 @@ You can see the live demo at: https://linkshub.vercel.app
...
 ## Getting Started 👩‍💻
 
 > ⚠️Prerequisites
-> 
-> * Before getting into it make you sure you have [npm](https://nodejs.org/download) installed.
-> 
-> * Also make sure to follow [CONTRIBUTING guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
-
+>
+> - Before getting into it make you sure you have [pnpm](https://nodejs.org/download) installed.
```
Comment (CBID2) [inline README.md:64]:
```suggestion
> - Before getting into, it make you sure you have [pnpm](https://nodejs.org/download) installed.
```

File: README.md [hunk 2]
```diff
@@ -51,74 +53,80 @@ You can see the live demo at: https://linkshub.vercel.app
...
+>
+> - Also make sure to follow [CONTRIBUTING guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
```
Comment (CBID2) [inline README.md:66]:
```suggestion
> - Also, make sure to follow [CONTRIBUTING guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
```

File: README.md [hunk 3]
```diff
@@ -2,7 +2,6 @@
 <img src="https://user-images.githubusercontent.com/78981177/215495029-ff9d4d24-a626-494a-859d-293cb9925f63.png"/>
```
Comment (CBID2) [inline README.md:2]:
```suggestion
<img src="https://user-images.githubusercontent.com/78981177/215495029-ff9d4d24-a626-494a-859d-293cb9925f63.png"/ alt=“”>
```

File: README.md [hunk 4]
```diff
@@ -2,7 +2,6 @@
...
 </div><hr>
```
Comment (CBID2) [inline README.md:3]:
```suggestion
</article><hr>
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @priyankarpal! 😄 I made some suggestions on your PR. In the future, consider using `<div>`tags sporadically and instead use semantic Html tags like `<article>`. It’ll help the code more accessible for people with disabilities that use screen readers. Other than that, well done! 😄

File: README.md (hunk 3 above)
Comment (rupali-codes) [reply README.md:2]: Better if we could add some text

**Comment by priyankarpal** [comment]
@CBID2 I resolved it as you said.

**Review by CBID2 — APPROVED** [review]
Well done @priyankarpal! 😀 LGTM! 👍

**Review by rupali-codes — COMMENTED** [review]
@priyankarpal shouldn't we remove package-lock.json file?

**Review by ujjawaltyagii — APPROVED** [review]
Looks good to me !! 👍

**Comment by rupali-codes** [comment]
Thanks @ujjawaltyagii

**Pull request merged**